swap_meet/clothing.py

```diff
@@ -1,2 +1,7 @@
-class Clothing:
-    pass
\ No newline at end of file
+from swap_meet.item import Item
+
+class Clothing(Item):
+    def __init__(self, age=None, condition=None, category="Clothing"):
+        super().__init__(age, condition, category)
```

Comment:
Nice work relying on the super class's constructor to initialize an instance of Clothing on line 5, like a helper. 

A quick note about the `category` parameter: from the README, for Wave 5 we want each specific type of Item to have a particular value for category. The Clothing class should have category "Clothing", which you've done by passing it in as a default value in the parameter list. 

However, since we're setting a default value like "Clothing", our intent is that we do not want the method caller to be able to set a different category. To really convey that intent, we can remove `category` altogether from the available initializer parameters and directly pass it to the super class constructor.

```python
def __init__(self, age=None, condition=None):
        super().__init__(age, condition, "Clothing")
```

The caller could always change the category by assigning directly to it after:

```python
clothing = Clothing(condition=3)
clothing.category = "Electronics"
```

But at least we're indicating that they shouldn't do that when we organize the code this way.